Sources/swift-format/Subcommands/LintFormatOptions.swift

```diff
@@ -108,7 +108,7 @@ struct LintFormatOptions: ParsableArguments {
   var experimentalFeatures: [String] = []
 
   /// The list of paths to Swift source files that should be formatted or linted.
-  @Argument(help: "Zero or more input filenames.")
+  @Argument(help: "Zero or more input filenames. If no paths are provided, stdin input will be used (use `-`).")
```

Comment:
```suggestion
  @Argument(help: "Zero or more input filenames. Use `-` for stdin.")
```

No reason to explicitly document the "no paths" behavior if we're planning to deprecate it in the future; let's just document the happy path.